src/app/list/tree-list/tree-list.component.ts

```diff
@@ -171,6 +168,6 @@ export class TreeListComponent extends ListBase implements DoCheck, OnInit {
    * @returns {string} The indent in pixels
    */
   private getIndentChildren(nodeLevel: number): string {
-    return (nodeLevel > 1) ? (nodeLevel - 1) * this.config.indentChildren + '' : '';
+    return nodeLevel > 1 ? (nodeLevel - 1) * this.config.indentChildren + '' : '';
```

Comment:
Not sure why the parens must be removed when Prettier has added them in other places?

Reply:
This is again and opinionated convention thing with Prettier we'd not currently be able to change...